.github/workflows/ci.yml

```diff
@@ -305,6 +305,16 @@ jobs:
     if: needs.detect-noop.outputs.noop != 'true'
 
     steps:
+      - name: Cleanup Disk
+        uses: jlumbroso/free-disk-space@main
```

Comment:
@phisco just a hint that, as discussed in https://github.com/crossplane-contrib/provider-aws/pull/1888#discussion_r1350594005, this always defaults to what is in the `main` and might better be pinned.

Reply:
Oh, I was expecting renovate to propose that change after merge 🤔 good catch!